tests/test_free_energy.py

```diff
@@ -170,14 +186,31 @@ def vacuum_model(ff_params):
         client = CUDAPoolClient(1)
         box = np.eye(3, dtype=np.float64)*100
 
+        harmonic_bond_potential = unbound_potentials[0]
+        group_idxs = get_group_indices(get_bond_list(harmonic_bond_potential))
+
+        x0 = coords
+        v0 = np.zeros_like(coords)
+        client = CUDAPoolClient(1)
+        temperature = 300.0
+        pressure = 1.0
+
         integrator = LangevinIntegrator(
-            300.0,
+            temperature,
             1.5e-3,
             1.0,
             masses,
             seed
         )
 
+        barostat = MonteCarloBarostat(
+            x0.shape[0],
+            pressure,
+            temperature,
+            group_idxs,
+            25,
```

Comment:
We may want to use a higher frequency during equilibration (eg. 5), and use only 25 for production.